sup3r/utilities/era_downloader.py

```diff
@@ -0,0 +1,1016 @@
+"""Download ERA5 file for the given year and month
+
+NOTE: To use this you need to have cdsapi package installed and a ~/.cdsapirc
+file with a url and api key.  Follow the instructions here:
+https://cds.climate.copernicus.eu/api-how-to
+"""
+
+import logging
+import os
+from calendar import monthrange
+from concurrent.futures import (
+    ProcessPoolExecutor,
+    ThreadPoolExecutor,
+    as_completed,
+)
+from glob import glob
+from warnings import warn
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from netCDF4 import Dataset
+
+from sup3r.utilities.log_interpolator import LogLinInterpolator
+
+logger = logging.getLogger(__name__)
+
+try:
+    import cdsapi
+    c = cdsapi.Client()
```

Comment:
Global single-char variable? bad idea. Recommend `CDS_API_CLIENT` (all caps for global, multiple chars because easier to search and find)